app/views/unattended/provisioning_templates/iPXE/ipxe_global_default.erb

```diff
@@ -0,0 +1,47 @@
+<%#
+kind: iPXE
+model: ProvisioningTemplate
+name: iPXE global default
+snippet: false
+-%>
+#!ipxe
+
+set menu-default local
+set menu-timeout 5000
+
+:start
+menu iPXE global boot menu
+item --key l local     Continue local boot
+item shell             Drop into iPXE shell
+item reboot            Reboot system
+item
+item --key d discovery Foreman Discovery
+choose --timeout ${menu-timeout} --default ${menu-default} selected || goto cancel
+set menu-timeout 0
+goto ${selected}
+
+:cancel
+echo Menu canceled, dropping to shell
+
+:shell
+echo Use the command 'exit' to return to menu
+shell
+set menu-timeout 0
```

Comment:
Why zero? This will disable the countdown completely? Not following, also there's the same line above.

Reply:
Most other PXE systems seem to act in similar manners, where the default boot timeout is aborted once you start interacting with the menu. Dropping into grub shell and then back to menu won't resume the default boot countdown as far as I recall.

Reply:
Okay.